src/App/Pages/Settings/OptionsPageViewModel.cs

```diff
@@ -166,6 +190,8 @@ public async Task InitAsync()
             DisableFavicon = (await _stateService.GetDisableFaviconAsync()).GetValueOrDefault();
             var theme = await _stateService.GetThemeAsync();
             ThemeSelectedIndex = ThemeOptions.FindIndex(k => k.Key == theme);
+            var autoDarkTheme = await _stateService.GetAutoDarkThemeAsync() ?? "dark";
```

Comment:
I'd really like if theme strings (like "dark", "nord", and so on) would be consts (or an Enum even better) in `ThemeManager` that can be referenced. So that there is less chance of misspelling and if for some reason we change any of them it's easier to replace.
Maybe not for this PR, but we should keep this in mind.
What do you think?